src/main/java/carpet/script/CarpetExpression.java

```diff
@@ -2899,6 +2900,51 @@ private void API_AuxiliaryAspects()
             return (cc, tt) -> time;
         });
 
+        this.expr.addLazyFunction("unix_time",0,(c,t,lv)->{
```

Comment:
Yeah - these need to go to Expression.java, not CarpetExpression, since they have nothing to do with minecraft.

Reply:
in Expression section you will have better functions including non-lazy equivalents where arguments come already evaluated.

Reply:
ok, done